src/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.sol

```diff
@@ -592,4 +592,29 @@ contract FM_BC_Bancor_Redeeming_VirtualSupply_v1 is
             );
         }
     }
+
+    //--------------------------------------------------------------------------
+    // Overriden Internal Functions
+
+    /// @dev    Distributes issuance tokens by minting them to the receiver.
+    /// @param  _receiver The address that will receive the bought tokens.
+    /// @param  _issuanceMintAmount The amount of issuance tokens to distribute.
+    function _distributeIssuanceToken(
+        address _receiver,
+        uint _issuanceMintAmount //@todo test
+    ) internal virtual override {
+        // Mint tokens to address
```

Comment:
I like comments, but I think our policy should be: add comments if additional context would be helpful and they would provide any form of additional information. 

I don't think this comment really helps here, as even a very non-technical person would know that `_mint(...)` signals that something is minted :D Just came to my mind now, we can probably talk about this elsewhere. But this is a bit superfluous imho.

Reply:
Aah Supermaven lets go
Yeah you are correct I will remove it

Reply:
Done